config/config-ingressgateway.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-ingressgateway
```

Comment:
I'm worried that this name is overly specific, perhaps we could call this `config-istio` and have it hold the istio-specific concepts that our `networking` controllers back by istio need?

Reply:
+1 to having one config map for Istio related settings.  I think there are a few Istio items in `config-networking` that could move here as well (in a later PR).

Reply:
done